### DISCUSSION TITLE
v10 Disable intro animation

### DISCUSSION THREAD

**Discussion by zRelux** [rnmapbox/maps#2211] [General]
Every time I open the app there is an animation in the map. How do I disable it?

**Comment by Gnative** [comment]
For me this started with the last update to Camera and follow user settings. 

https://github.com/rnmapbox/maps/issues/2178

**Comment by mfazekas** [comment]
Camera has a defaultSettings that will not animate

**Reply by Gnative** [reply]
I should probibly have asked for more information.

If the Camera component is used and if any of the followUser props are set the map animates. This might be the cause of what is being described here.

**Reply by statico** [reply]
I'm on ^10.0.11 and I still see the loading animation that spins the entire earth from 0,0 to the user's location:

```
      <Mapbox.MapView
        ref={map}
        style={styles.map}
        scaleBarEnabled={false}
        rotateEnabled={false}
        logoEnabled={false}
        onPress={hideSelectedCard}
        onCameraChanged={handleCameraChanged}
      >
        <Mapbox.Camera
          ref={camera}
          animationMode="moveTo"
          animationDuration={1}
          followUserLocation={false}
        />
        ```

**Reply by statico** [reply]
Ah, actually this was because i was using `Camera.setCamera`. Configuring that method call with animation parameters helps minimize the transition.
